### PR TITLE
Added support to v3.4.0 of geonetwork

### PR DESCRIPTION
**Pull request by doublebyte1**
- Fixed commit hash of GN 3.2.
- Added support for GN 3.4.

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 79652be..f21e3b0 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,4 +1,6 @@
 geonetwork:3.0 @ amd64
 geonetwork:3.0-postgres @ amd64
+geonetwork:3.4.0 @ amd64
+geonetwork:3.4.0-postgres @ amd64
 geonetwork:latest @ amd64
 geonetwork:postgres @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 1280308..65143de 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -6,5 +6,7 @@ geonetwork:3.2
 geonetwork:3.2-postgres
 geonetwork:3.2.2
 geonetwork:3.2.2-postgres
+geonetwork:3.4.0
+geonetwork:3.4.0-postgres
 geonetwork:latest
 geonetwork:postgres
diff --git a/geonetwork_postgres/Dockerfile b/geonetwork_3.4.0-postgres/Dockerfile
similarity index 97%
copy from geonetwork_postgres/Dockerfile
copy to geonetwork_3.4.0-postgres/Dockerfile
index 3c9458c..691387d 100644
--- a/geonetwork_postgres/Dockerfile
+++ b/geonetwork_3.4.0-postgres/Dockerfile
@@ -1,4 +1,4 @@
-FROM geonetwork:3.2.2
+FROM geonetwork:3.4.0
 
 RUN apt-get update && apt-get install -y postgresql-client && \
     rm -rf /var/lib/apt/lists/*
diff --git a/geonetwork_3.0-postgres/docker-entrypoint.sh b/geonetwork_3.4.0-postgres/docker-entrypoint.sh
similarity index 100%
copy from geonetwork_3.0-postgres/docker-entrypoint.sh
copy to geonetwork_3.4.0-postgres/docker-entrypoint.sh
diff --git a/geonetwork_3.0-postgres/jdbc.properties b/geonetwork_3.4.0-postgres/jdbc.properties
similarity index 100%
copy from geonetwork_3.0-postgres/jdbc.properties
copy to geonetwork_3.4.0-postgres/jdbc.properties
diff --git a/geonetwork_latest/Dockerfile b/geonetwork_3.4.0/Dockerfile
similarity index 82%
copy from geonetwork_latest/Dockerfile
copy to geonetwork_3.4.0/Dockerfile
index 23c87b7..0e1a3d7 100644
--- a/geonetwork_latest/Dockerfile
+++ b/geonetwork_3.4.0/Dockerfile
@@ -2,11 +2,11 @@ FROM tomcat:8.0-jre8
 
 ENV GN_FILE geonetwork.war
 ENV DATA_DIR=$CATALINA_HOME/webapps/geonetwork/WEB-INF/data
-ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:PermSize=512m -XX:MaxPermSize=1024m -XX:+UseConcMarkSweepGC"
+ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:+UseConcMarkSweepGC"
 
 #Environment variables
-ENV GN_VERSION 3.2.2
-ENV GN_DOWNLOAD_MD5 93b5279223503106ffeea3e9ca697805
+ENV GN_VERSION 3.4.0
+ENV GN_DOWNLOAD_MD5 f7d274a5f49d5f0968b204d290308943
 
 WORKDIR $CATALINA_HOME/webapps
 
diff --git a/geonetwork_3.0/docker-entrypoint.sh b/geonetwork_3.4.0/docker-entrypoint.sh
similarity index 100%
copy from geonetwork_3.0/docker-entrypoint.sh
copy to geonetwork_3.4.0/docker-entrypoint.sh
diff --git a/geonetwork_latest/Dockerfile b/geonetwork_latest/Dockerfile
index 23c87b7..91cb5cb 100644
--- a/geonetwork_latest/Dockerfile
+++ b/geonetwork_latest/Dockerfile
@@ -2,7 +2,7 @@ FROM tomcat:8.0-jre8
 
 ENV GN_FILE geonetwork.war
 ENV DATA_DIR=$CATALINA_HOME/webapps/geonetwork/WEB-INF/data
-ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:PermSize=512m -XX:MaxPermSize=1024m -XX:+UseConcMarkSweepGC"
+ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:+UseConcMarkSweepGC"
 
 #Environment variables
 ENV GN_VERSION 3.2.2
```

</details>

File: library/geonetwork [hunk 1]
```diff
@@ -12,9 +12,17 @@ GitCommit: e5cf5da76f557481c35b4a4d3e3cac77768a1302
...
 Tags: 3.2.2, 3.2, latest
-GitCommit: 7ba5f533d9ee31cf1a22d4d6a1a84fbb74f86466
+GitCommit: a9d9f3f07c0f2b8badadb74206647d7f6dea4b24
 Directory: 3.2.2
 
 Tags: 3.2.2-postgres, 3.2-postgres, postgres
 GitCommit: 7ba5f533d9ee31cf1a22d4d6a1a84fbb74f86466
 Directory: 3.2.2/postgres
+
+Tags: 3.4.0
```
Comment (yosifkit) [inline library/geonetwork]: Did you want to move `latest` to `3.4.0` and have a `3.4` alias (and move `postgres` and add `3.4-postgres` below)?

Comment (doublebyte1) [reply library/geonetwork]: Yes, thanks for pointing that out. I have fixed that in my latest [commit](https://github.com/docker-library/official-images/pull/3792/commits/fb6c4bfddb5a131017e0fa2f0cd732c0ea25514d).

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 79652be..75b6afa 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,4 +1,6 @@
 geonetwork:3.0 @ amd64
 geonetwork:3.0-postgres @ amd64
+geonetwork:3.2 @ amd64
+geonetwork:3.2-postgres @ amd64
 geonetwork:latest @ amd64
 geonetwork:postgres @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 1280308..88e00e8 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -6,5 +6,9 @@ geonetwork:3.2
 geonetwork:3.2-postgres
 geonetwork:3.2.2
 geonetwork:3.2.2-postgres
+geonetwork:3.4
+geonetwork:3.4-postgres
+geonetwork:3.4.0
+geonetwork:3.4.0-postgres
 geonetwork:latest
 geonetwork:postgres
diff --git a/geonetwork_postgres/Dockerfile b/geonetwork_3.2-postgres/Dockerfile
similarity index 100%
copy from geonetwork_postgres/Dockerfile
copy to geonetwork_3.2-postgres/Dockerfile
diff --git a/geonetwork_3.0-postgres/docker-entrypoint.sh b/geonetwork_3.2-postgres/docker-entrypoint.sh
similarity index 100%
copy from geonetwork_3.0-postgres/docker-entrypoint.sh
copy to geonetwork_3.2-postgres/docker-entrypoint.sh
diff --git a/geonetwork_3.0-postgres/jdbc.properties b/geonetwork_3.2-postgres/jdbc.properties
similarity index 100%
copy from geonetwork_3.0-postgres/jdbc.properties
copy to geonetwork_3.2-postgres/jdbc.properties
diff --git a/geonetwork_latest/Dockerfile b/geonetwork_3.2/Dockerfile
similarity index 90%
copy from geonetwork_latest/Dockerfile
copy to geonetwork_3.2/Dockerfile
index 23c87b7..91cb5cb 100644
--- a/geonetwork_latest/Dockerfile
+++ b/geonetwork_3.2/Dockerfile
@@ -2,7 +2,7 @@ FROM tomcat:8.0-jre8
 
 ENV GN_FILE geonetwork.war
 ENV DATA_DIR=$CATALINA_HOME/webapps/geonetwork/WEB-INF/data
-ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:PermSize=512m -XX:MaxPermSize=1024m -XX:+UseConcMarkSweepGC"
+ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:+UseConcMarkSweepGC"
 
 #Environment variables
 ENV GN_VERSION 3.2.2
diff --git a/geonetwork_3.0/docker-entrypoint.sh b/geonetwork_3.2/docker-entrypoint.sh
similarity index 100%
copy from geonetwork_3.0/docker-entrypoint.sh
copy to geonetwork_3.2/docker-entrypoint.sh
diff --git a/geonetwork_latest/Dockerfile b/geonetwork_latest/Dockerfile
index 23c87b7..0e1a3d7 100644
--- a/geonetwork_latest/Dockerfile
+++ b/geonetwork_latest/Dockerfile
@@ -2,11 +2,11 @@ FROM tomcat:8.0-jre8
 
 ENV GN_FILE geonetwork.war
 ENV DATA_DIR=$CATALINA_HOME/webapps/geonetwork/WEB-INF/data
-ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:PermSize=512m -XX:MaxPermSize=1024m -XX:+UseConcMarkSweepGC"
+ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:+UseConcMarkSweepGC"
 
 #Environment variables
-ENV GN_VERSION 3.2.2
-ENV GN_DOWNLOAD_MD5 93b5279223503106ffeea3e9ca697805
+ENV GN_VERSION 3.4.0
+ENV GN_DOWNLOAD_MD5 f7d274a5f49d5f0968b204d290308943
 
 WORKDIR $CATALINA_HOME/webapps
 
diff --git a/geonetwork_postgres/Dockerfile b/geonetwork_postgres/Dockerfile
index 3c9458c..691387d 100644
--- a/geonetwork_postgres/Dockerfile
+++ b/geonetwork_postgres/Dockerfile
@@ -1,4 +1,4 @@
-FROM geonetwork:3.2.2
+FROM geonetwork:3.4.0
 
 RUN apt-get update && apt-get install -y postgresql-client && \
     rm -rf /var/lib/apt/lists/*
```

</details>

**Comment by yosifkit** [comment]
Build test of #3792; fb6c4bfd; `amd64` (`geonetwork`):

```console

$ bashbrew build geonetwork:3.0.5
Using bashbrew/cache:32da04588cf56657a8e5f5d949b4aab06bf620d2b77b6961bcb2dbfd1afa4382 (geonetwork:3.0.5)
Tagging geonetwork:3.0.5
Tagging geonetwork:3.0

$ test/run.sh geonetwork:3.0.5
testing geonetwork:3.0.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.0.5-postgres
Using bashbrew/cache:18b5705e6980f25306ce8f01e4a031f04b22b2640968050e5e49d866d5ec9c74 (geonetwork:3.0.5-postgres)
Tagging geonetwork:3.0.5-postgres
Tagging geonetwork:3.0-postgres

$ test/run.sh geonetwork:3.0.5-postgres
testing geonetwork:3.0.5-postgres
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.2.2
Using bashbrew/cache:de44a3ad126c4cd3c32fafdf8be63314f9933f92e2d4a2d507be676439c89994 (geonetwork:3.2.2)
Tagging geonetwork:3.2.2
Tagging geonetwork:3.2

$ test/run.sh geonetwork:3.2.2
testing geonetwork:3.2.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.2.2-postgres
Using bashbrew/cache:bffb359a41fec2e4d1b0f9cb59c41c5ed49690044274789f6a02fc92cfcd1757 (geonetwork:3.2.2-postgres)
Tagging geonetwork:3.2.2-postgres
Tagging geonetwork:3.2-postgres

$ test/run.sh geonetwork:3.2.2-postgres
testing geonetwork:3.2.2-postgres
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.4.0
Using bashbrew/cache:fb52f1d7e9d12f15c898316004447cd700604f438d8a289ee8856c49306c5080 (geonetwork:3.4.0)
Tagging geonetwork:3.4.0
Tagging geonetwork:3.4
Tagging geonetwork:latest

$ test/run.sh geonetwork:3.4.0
testing geonetwork:3.4.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.4.0-postgres
Using bashbrew/cache:1b0648501d4b4316c011f793b3509d3a44801eff651c7791ed75d6ba47ddaba3 (geonetwork:3.4.0-postgres)
Tagging geonetwork:3.4.0-postgres
Tagging geonetwork:3.4-postgres
Tagging geonetwork:postgres

$ test/run.sh geonetwork:3.4.0-postgres
testing geonetwork:3.4.0-postgres
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**